### PR TITLE
Foxy branch for moveit2 tutorial on foxy

### PR DESCRIPTION
**Pull request by tylerjw**
### Description

MoveIt2 now has a foxy branch.  The tutorials for foxy should be updated to avoid confusing users.

### CODE REVIEW

File: doc/getting_started/getting_started.rst [hunk 1]
```diff
@@ -34,7 +34,7 @@ You will need to have a `colcon <https://docs.ros.org/en/foxy/Tutorials/Colcon-T
   mkdir -p ~/ws_moveit2/src
   cd ~/ws_moveit2/src
 
-  git clone https://github.com/ros-planning/moveit2.git
+  git clone https://github.com/ros-planning/moveit2.git -b foxy
```
Comment (vatanaksoytezer) [inline doc/getting_started/getting_started.rst:37]: Should we add a comment that we also support Rolling and Galactic, and ros2 branches should be used in that case?

Comment (vatanaksoytezer) [reply doc/getting_started/getting_started.rst:37]: @tylerjw

Comment (tylerjw) [reply doc/getting_started/getting_started.rst:37]: Yes, I think we could rethink the recommendations on this page entirely and encourage people using foxy to just use the debains of moveit.  If they want to develop on moveit they should probably be using rolling.

Comment (vatanaksoytezer) [reply doc/getting_started/getting_started.rst:37]: I'll update the instructions here on website today (latest tomorrow)

Comment (tylerjw) [reply doc/getting_started/getting_started.rst:37]: I'll close this PR then.

Comment (lorepieri8) [reply doc/getting_started/getting_started.rst:37]: @vatanaksoytezer I don't see the "-b foxy" change reflected in the website: http://moveit2_tutorials.picknik.ai/doc/getting_started/getting_started.html

Comment (vatanaksoytezer) [reply doc/getting_started/getting_started.rst:37]: @lorepieri8 thanks for the heads up. PR is up to update this in #103. After https://github.com/ros-planning/moveit.ros.org/pull/623 gets merged everything will point moveit.ros.org which will contain the official build instructions. If you have any troubles with tutorials or source building or installation, feel free to open an issue and ping me. All the PRs should get merged pretty soon hopefully.

**Pull request closed without merging**